sdks/python/apache_beam/io/filebasedsource_test.py

```diff
@@ -44,18 +45,21 @@ def read_records(self, file_name, range_tracker):
     f = self.open_file(file_name)
     try:
       start = range_tracker.start_position()
-      f.seek(start)
+      if start > 0:
+        f.seek(start)
```

Comment:
Can you combine this with the clause below and just do f.seek(start - 1)?  Please add a comment in that case.